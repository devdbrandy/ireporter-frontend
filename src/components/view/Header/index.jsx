import React from 'react';
import NavBar from '../NavBar';

const Header = () => (
  <>
    <NavBar />
  </>
);

export default Header;
