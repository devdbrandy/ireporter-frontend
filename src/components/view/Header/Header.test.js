import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('<Header />', () => {
  it('renders without crashing given the required props', () => {
    const enzymeWrapper = shallow(<Header />);
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
