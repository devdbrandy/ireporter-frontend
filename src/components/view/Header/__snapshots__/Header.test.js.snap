// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing given the required props 1`] = `
<Fragment>
  <NavBar />
</Fragment>
`;
